content/news/2021-08-19-gsoc-tools.md

```diff
@@ -1,16 +1,17 @@
 +++
-title = "Devin's Google Summer of Code Ending"
+title = "GSoC 2021: Tooling Final Summary"
 date = 2021-08-19
 extra.author = "Devin Ragotzy"
 +++
 
-# A Summer of Ruma Tooling Comes to an End
+# Google Summer of Code Comes to an End for Ruma Tooling
```

Comment:
There shouldn't be any level-1 headings inside the article, that's used for the title.